### PR TITLE
[10.0][UPD] shopinvader cart expiry: apply only on draft cart

### PR DESCRIPTION
**Pull request by acsonefho**
The cart expiry should be applied only on draft cart.

### CODE REVIEW

File: shopinvader_cart_expiry/tests/test_cart_expiry.py [hunk 1]
```diff
@@ -68,3 +68,32 @@ def test_cart_expiry_delete(self):
...
+        :return:
+        """
+        so_date = fields.Datetime.from_string(self.sale.write_date)
+        today = fields.Datetime.to_string(so_date + timedelta(hours=5))
+        self.sale.write({"state": "sent"})
+        self.backend.write(
+            {"cart_expiry_delay": 1, "cart_expiry_policy": "cancel"}
+        )
+        now_method = "odoo.fields.Datetime.now"
+        with mock.patch(now_method) as mock_now:
```
Comment (simahawk) [inline shopinvader_cart_expiry/tests/test_cart_expiry.py:84]: yet another test mocking dates as https://github.com/shopinvader/odoo-shopinvader/pull/523#discussion_r353072199

Please, introduce freezegun :pray: 
CC @lmignon

**Review by simahawk — CHANGES_REQUESTED** [review]

File: shopinvader_cart_expiry/tests/test_cart_expiry.py (hunk 1 above)
Comment (lmignon) [reply shopinvader_cart_expiry/tests/test_cart_expiry.py:84]: I fully agree with @simahawk [freezegun](https://pypi.org/project/freezegun/) is so powerful  and so beautiful syntactically

Comment (acsonefho) [reply shopinvader_cart_expiry/tests/test_cart_expiry.py:84]: @simahawk @lmignon I totally agree: it seems more readable and easy to use and understand.
As existing tests (into this same file) are using `mock`, it'll be strange to use 2 different ways to test something quite similar. And the purpose of this MR is not to improve existing tests.
So I propose to do this improvement into a new MR.

**Pull request merged**